std_msgs/Header header
ObjectPose[] objects
geometry_msgs/Pose reference_pose